content/en/docs/setup/independent/high-availability.md

```diff
@@ -266,81 +266,87 @@ Please select one of the tabs to see installation instructions for the respectiv
-          image: k8s.gcr.io/etcd-amd64:3.1.10
-          livenessProbe:
-            httpGet:
-              path: /health
-              port: 2379
-              scheme: HTTP
-            initialDelaySeconds: 15
-            timeoutSeconds: 15
-          name: etcd
-          env:
-          - name: PUBLIC_IP
-            valueFrom:
-              fieldRef:
-              fieldPath: status.hostIP
-          - name: PRIVATE_IP
-            valueFrom:
-              fieldRef:
-                fieldPath: status.podIP
-          - name: PEER_NAME
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.name
-          volumeMounts:
-          - mountPath: /var/lib/etcd
-            name: etcd
-          - mountPath: /certs
-            name: certs
-      hostNetwork: true
-      volumes:
-      - hostPath:
-          path: /var/lib/etcd
-          type: DirectoryOrCreate
-        name: etcd
-      - hostPath:
-          path: /etc/kubernetes/pki/etcd
-        name: certs
-      EOF
+```none
+cat >/etc/kubernetes/manifests/etcd.yaml <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    component: etcd
+    tier: control-plane
+  name: <podname>
+  namespace: kube-system
+spec:
+  containers:
+  - command:
+    - etcd --name <name>
+    - --data-dir /var/lib/etcd
+    - --listen-client-urls http://localhost:2379
+    - --advertise-client-urls http://localhost:2379
+    - --listen-peer-urls http://localhost:2380
+    - --initial-advertise-peer-urls http://localhost:2380
+    - --cert-file=/certs/server.pem
+    - --key-file=/certs/server-key.pem
+    - --client-cert-auth
+    - --trusted-ca-file=/certs/ca.pem
+    - --peer-cert-file=/certs/peer.pem
+    - --peer-key-file=/certs/peer-key.pem
+    - --peer-client-cert-auth
+    - --peer-trusted-ca-file=/certs/ca.pem
+    - --initial-cluster etcd0=https://<etcd0-ip-address>:2380,etcd1=https://<etcd1-ip-address>:2380,etcd2=https://<etcd2-ip-address>:2380
+    - --initial-cluster-token my-etcd-token
+    - --initial-cluster-state new
+    image: k8s.gcr.io/etcd-amd64:3.1.10
+    livenessProbe:
+      httpGet:
+        path: /health
+        port: 2379
+        scheme: HTTP
+      initialDelaySeconds: 15
+      timeoutSeconds: 15
+    name: etcd
+    env:
+    - name: PUBLIC_IP
+      valueFrom:
+        fieldRef:
+        fieldPath: status.hostIP
+    - name: PRIVATE_IP
+      valueFrom:
+        fieldRef:
+          fieldPath: status.podIP
+    - name: PEER_NAME
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.name
+    volumeMounts:
+    - mountPath: /var/lib/etcd
+      name: etcd
+    - mountPath: /certs
+      name: certs
+hostNetwork: true
+volumes:
+- hostPath:
+    path: /var/lib/etcd
+    type: DirectoryOrCreate
+  name: etcd
+- hostPath:
+    path: /etc/kubernetes/pki/etcd
+  name: certs
+EOF
+```
 
 Make sure you replace:
-* `<podname>` with the name of the node you're running on (e.g. `etcd0`, `etcd1` or `etcd2`)
```

Comment:
This was rendering as:

![image](https://user-images.githubusercontent.com/583147/41227287-3ecbfbec-6d42-11e8-8273-28076dc3fd14.png)

Reply:
After:
![image](https://user-images.githubusercontent.com/583147/41227836-378fd57c-6d44-11e8-9702-6e0d37d179f5.png)